raft/log.go

```diff
@@ -455,6 +482,14 @@ func (l *Log) Initialize() error {
 		return err
 	}
 
+	// Wait until leader state change.
```

Comment:
This comment doesn't seem to be quite right. It seems more like "wait until this node becomes leader", not a change in the state *of* the leader.